packages/shopify-app-remix/docs/i18n.md

```diff
@@ -0,0 +1,143 @@
+# Internationalization (i18n)
+
+This package includes some tools to help make it easier for apps to translate their interfaces.
+To do that, you'll need to perform the following steps:
+
+1. Install [`remix-i18next`](https://www.npmjs.com/package/remix-i18next) and [`react-i18next`](https://www.npmjs.com/package/react-i18next)
+1. Set up your language detector and backend, for example following [the i18next sample](https://www.i18next.com/overview/getting-started#extended-sample).
+1. Set up the i18next server, and init your client and server in your entrypoints.
+
+The utilities provided by this package for i18n are all in the `@shopify/shopify-app-remix/i18n` export path.
+
```

Comment:
This is fairly involved docs... unavoidable, but I wonder if a section encourage users to _just use the template_ would be useful.  E.g: Link to the template where it's already all setup.